packages/xo-server/src/api/vif.js

```diff
@@ -85,6 +85,13 @@ export async function set({
     push.apply(newIpAddresses, allowedIpv6Addresses || vif.allowedIpv6Addresses)
   }
 
+  if (lockingMode !== undefined) {
+    await this.checkPermissions(this.user.id, [
+      [vif.$VM, 'operate'],
```

Comment:
Isn't the permission check on the VIF itself enough?

/cc @badrAZ https://github.com/vatesfr/xen-orchestra/pull/5283#discussion_r505530630